test/test.js

```diff
@@ -27,14 +27,6 @@ describe('broccoli-uglify-sourcemap', function() {
     expect(builder.read()).toMatchSnapshot();
   });
 
-  it('generates expected output async', async function() {
-    builder = createBuilder(new Uglify(fixtures, { async: true }));
-
-    await builder.build();
-
-    expect(builder.read()).toMatchSnapshot();
-  });
```

Comment:
Hmm, why remove this test? I _think_ we probably want to keep it around...

Reply:
But, this test would probably fail since passing the `async` property is deprecated. I think it is better to write a separate [test case](https://github.com/ember-cli/broccoli-uglify-sourcemap/pull/237/files#diff-c1129c8b045390789fa8ff62f2c6b4a9R161) that addresses this deprecation error.  _Correct me If I misunderstood your comment?_

Reply:
Ah! The thing I missed was the test _just above_ this that was testing the same thing (the fixtures directory) using the default option values (and therefore avoiding the new error).

Sorry about that confusion!